src/agent/onefuzz-agent/src/tasks/regression/libfuzzer.rs

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use crate::tasks::{
+    config::CommonConfig,
+    report::{
+        crash_report::{CrashReport, CrashTestResult},
+        libfuzzer_report,
+    },
+    utils::default_bool_true,
+};
+
+use anyhow::Result;
+use reqwest::Url;
+
+use super::regression::{self, RegressionHandler};
+use async_trait::async_trait;
+use onefuzz::syncdir::SyncedDir;
+use serde::Deserialize;
+use std::{collections::HashMap, path::PathBuf};
+
+#[derive(Debug, Deserialize)]
+pub struct Config {
+    pub target_exe: PathBuf,
+
+    #[serde(default)]
+    pub target_options: Vec<String>,
+
+    #[serde(default)]
+    pub target_env: HashMap<String, String>,
+
+    pub inputs: Option<SyncedDir>,
+
+    pub input_reports: Option<SyncedDir>,
+    pub crashes: Option<SyncedDir>,
+    pub report_list: Vec<String>,
+
+    pub no_repro: SyncedDir,
+    pub reports: SyncedDir,
+
+    pub target_timeout: Option<u64>,
+
+    #[serde(default)]
```

Comment:
libfuzzer regression doesn't need check_asan_log or check_debugger.  However, it does need check_fuzzer_help.